### PR TITLE
Fix serializer attempting to get attributes from null object

### PR DESCRIPTION
**Pull request by joeljames**
@petebrowne @reneric @thatgibbyguy 

This PR fixes this issue https://github.com/localmed/pyserializer/issues/16

### CODE REVIEW

File: tests/functional_tests/pyserializer/serialization_tests.py [hunk 1]
```diff
@@ -199,3 +209,106 @@ def test_serialization_with_nested_source(self):
...
+        self.User = User
+        self.Comment = Comment
+
+    def test_serializer_when_nested_obj_is_none(self):
+        user = None
+        comment = self.Comment(user=user)
+        serializer = self.CommentSerializer(comment)
+        expected_output = {
+            'content': 'Some text content',
+            'location': None
```
Comment (petebrowne) [inline tests/functional_tests/pyserializer/serialization_tests.py:265]: 👍 Nice - this will be helpful outside of the original bug

**Comment by petebrowne** [comment]
👍

**Comment by reneric** [comment]
+1 Approve +1 Approve

**Comment by reneric** [comment]
+1 Approve +1 Approve +1 Approve

**Comment by reneric** [comment]
+1 Approve

**Comment by thatgibbyguy** [comment]
👍

**Pull request merged**